docs/ui-components/code.md

```diff
@@ -58,6 +58,8 @@ var fun = function lang(l) {
 ```
 {% endhighlight %}
 
+Syntax highlighting, line numbers, and HTML compression do not work together; **the combination of these features generates invalid HTML**. To learn more, see ["Code with line numbers"]({% link docs/ui-components/line-nos.md %}).
```

Comment:
Although this is true, it suggests that users can get valid HTML for highlighting with line numbers simply by turning off the compression. Doing that produces HTML which will probably render as intended, but is still invalid.

Perhaps replace "generates invalid HTML" by "generates invalid HTML that renders incorrectly"?